app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +137,35 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  const lowerBoundNumber = lowerBound?.abs() !== undefined && Number(lowerBound.abs())
```

Comment:
I think just lowerBound !== undefined should work, also casting BigNumber to Number directly is dangerous and can often lead to precision errors.
Use formatBigNumber instead with a precision value of 2 and then cast result to Number type.